fire/SFMainMod.F90

```diff
@@ -28,6 +28,21 @@ module SFMainMod
   use EDtypesMod            , only : NFSC
   use EDtypesMod            , only : TR_SF
 
+  use PRTGenericMod,          only : leaf_organ
+  use PRTGenericMod,          only : all_carbon_species
```

Comment:
Again unsure about the need for N,P species, etc. Is it a placeholder?

Reply:
These don't need to be listed.  Will remove.